boards/build_scripts/src/lib.rs

```diff
@@ -2,6 +2,6 @@
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 // Copyright Tock Contributors 2024.
 
-mod default;
+pub mod default;
```

Comment:
Do we want to expose this? We could namespace new functionality in the future, but what we have seems like the normal APIs we can just expose directly.

Reply:
Up to you -- I am fine to re-export all of the functions inside default, but it seems easier to just have default be pub and then we can still hide functions inside the file by making individual functions not pub